.github/workflows/embedded.yml

```diff
@@ -46,26 +48,40 @@ jobs:
           EMBED_BINUTILS: OFF
           RUN_ALL_TESTS: 1
           RUNTIME_TEST_DISABLE: builtin.cgroup
+          DISTRO: bionic
+        - TYPE: Release
+          NAME: vanilla_llvm+clang+glibc2.23
```

Comment:
this is what i mean, i tihnk we only need the vanilla target, no need to fuss with the libclang one for older versions IMO.

Reply:
think that's what you meant? wasn't sure if you wanted to keep bionic + debian llvm builds?

Reply:
Ah sorry got this backwards.

I think the builds should track the newer distro (bionic, though maybe this should update also support disco or eoan since you've parameterized it?), but also net add the vanilla xenial build. So the build targets should be:

- Ubuntu 18.04 debian_llvm+libclang+glibc2.27 (Debug)
- Ubuntu 18.04 debian_llvm+libclang+glibc2.27 (Release)
- Ubuntu 18.04 vanilla_llvm+clang+glibc2.27 (Release) - Bionic
- Ubuntu 16.04 vanilla_llvm+clang+glibc2.23 (Release) - Xenial (the net-new target)

I checked out your branch on actions https://github.com/alexeldeib/bpftrace/runs/418369669?check_suite_focus=true and it looks like this just replaces the 2.27 glibc build, which I think we should keep.

Reply:
Ah, I was confused because I thought that's what I did originally. I'll add it back.

Let's get xenial hammered out for the static build and i'd be happy to update CI for disco, eoan, or focal in a follow up?

Reply:
My mistake I think, I think I interpreted that it would make 6 builds instead of 4 because of adding xenial debian+libclang. Thanks for making the change back haha

> Let's get xenial hammered out for the static build and i'd be happy to update CI for disco, eoan, or focal in a follow up?

Yeah in a separate PR i want to add another "edge" build, that should build off of the latest LLVM, ubuntu, BCC, etc, so that hopefully we can catch issues earlier in the release process of these various dependencies, and also offer a fully bleeding-edge build.